corehq/apps/export/templates/export/bootstrap5/datasource_export_view.html

```diff
@@ -13,15 +13,15 @@
 {% block page_content %}
   <p>
     {% blocktrans %}
-    Generate a query file for the Data Export Tool from any of the project's data sources.
+      Generate a query file for the Data Export Tool from any of the project's data sources.
```

Comment:
Instead of adding space, why don't we add margin or padding to the <p> ?

Reply:
Since there's already whitespace between the `<p>` and the starting `G`, this doesn't affect the UI. It's just for code indentation.

Reply:
I see! Thank you!